tests/example_9_nuclear_dynamics/aux_functs.py

```diff
@@ -96,32 +96,12 @@ def compute_etot(ham, p, iM):
     
     return Ekin, Epot, Etot
 
-def sample(x, mean_x, sigma_x, rnd, sample_opt):  
-    nr, nc = x.num_of_rows, x.num_of_cols
-
-    if sample_opt == 0:
-        for i in range(nr):
-            for j in range(nc):    
-                x.set(i,j, mean_x.get(i,0) + sigma_x.get(i,0) * rnd.normal() )
-
-    elif sample_opt == 1:
-        params = {"k":0.032, "m":2000.0, "states":[0], "coeffs":[1.0]}
-        sampling = metropolis_gau(rnd, HO_sup, x, params, 1000, 500, 0.25)
-
-        for i in range(nr):
-            for j in range(nc):
-                    x.set(i,j, sampling[j].get(0,i) + mean_x.get(i,0))
-
-    elif sample_opt == 2:
-        params = {"k":0.032, "m":2000.0, "states":[0], "coeffs":[1.0]}
-        sampling = metropolis_gau(rnd, HO_sup, x, params, 1000, 500, 2.0)
-
-        for i in range(nr):
-            for j in range(nc):
-                    x.set(i,j, sampling[j].get(0,i) + mean_x.get(i,0))
-
-
+def sample(x, mean_x, sigma_x, rnd):  

```

Comment:
this function is already defined in libra_py/tsh.py Name the function that you introduce in this module differently, so there is a minimal confusion on which function is which.